base_time_parameter/readme/USAGE.rst

```diff
@@ -1,21 +1,30 @@
-First give your user the access right "Manage Time Parameters".
-Then in "Settings/Technical/Time Parameters" you can create a new parameter
-with different versions (start date and value).
+Menu: "Settings/Technical/Time Parameters"
+
+Create a parameter with different versions (start date and value).
+
+If model_id is empty, any model/record may get the time parameter.
+
+The value may be a text or reference.
+
+Get the value like this:
 
-Then you can access current parameter value like this:
 .. code-block:: python
 
-    # using today's date when no date is passed
-    value = model.get_time_parameter('my_parameter_name')
-    # hr.payslip: use payslip.date_to
-    value = payslip.get_time_parameter('my_parameter_name', payslip.date_to)
-    # account.tax: use tax_date (https://github.com/apps2grow/apps/tree/14.0/account_tax_python_with_date)
-    tax_rate = float(company.get_time_parameter('tax_high_rate', tax_date))
+    # Pass no date: using today's date
+    value = model.get_time_parameter("parameter_code_or_name")
+    # Pass a date or datetime
+    value = model.get_time_parameter("parameter_code_or_name", date=datetime.datetime.now()))
+    # Pass the name of a date/datetime field of the record
+    value = record.get_time_parameter("parameter_code_or_name", "date")
+
+Example of implementation in another module
+-------------------------------------------
```

Comment:
You can't use this underlining, as it's used when the fragments are composed. Use instead `~~~~~~`.